api/ip.js

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
This module needs a `default` exports as all `socket:` modules require it:

```js
export default { isIPv4 }
```

should work